### PR TITLE
Refs #8710 - install katello-debug by default

### PR DESCRIPTION
**Pull request by lzap**
Belongs to https://github.com/Katello/katello/pull/4970

### CODE REVIEW

**Comment by lzap** [comment]
Hello, anyone? @iNecas @jlsherrill Thanks!

**Comment by lzap** [comment]
Context: I've created foreman-debug and katello-debug subpackages. We want to install them on the Server as well as on the Capsules. Katello-debug has dependency of foreman-debug (it is technically an "extension" to it).

**Comment by stbenjam** [comment]
Yea this is fine, but the package has to exist before we merge it, it's not in the nightly repos yet.

I imagine because the Pipeline is red? http://ci.theforeman.org/view/Katello%20Pipeline/

File: manifests/install.pp [hunk 1]
```diff
@@ -5,6 +5,10 @@
     package{ ['rubygem-smart_proxy_pulp']:
       ensure => installed,
     }
+
+    package{ ['katello-debug']:
+      ensure => installed,
```
Comment (stbenjam) [inline manifests/install.pp]: Actually why is this in the if block for pulp? We should always install `katello-debug`, or should we?

**Comment by iNecas** [comment]
What @stbenjam sais

**Comment by lzap** [comment]
Right fixed.

**Comment by stbenjam** [comment]
Looks like the packages are there now, so ACK

**Comment by stbenjam** [comment]
@lzap You can merge your own PR's, it's how it's done in the Katello GitHub org, but I'll merge this one

**Pull request merged**